hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/TagManagementTest.java

```diff
@@ -46,9 +47,27 @@
 @Stories("Tag Management")
 public class TagManagementTest extends AbstractJpaIntegrationTest {
 
-    @Before
-    public void setup() {
-        assertThat(targetTagRepository.findAll()).as("Not tags should be available").isEmpty();
+    private static final String NOT_EXIST_ID = "1234";
+
+    @Test
+    @Description("Verifies that management queries react as specfied on calls for non existing entities.")
+    public void nonExistingEntityQueries() {
```

Comment:
We also have these cases in other test methods were we want to make sure that on events are send. You can add ` @ExpectEvents({ @Expect(type = TargetTagCreatedEvent.class, count = 0) })` for example to you test method. This will also ensure that no events at all are send.